setup.py

```diff
@@ -65,6 +65,7 @@ def run_tests(self):
     version=about['__version__'],
     description=about['__description__'],
     long_description=readme + '\n\n' + history,
+    long_description_content_type='text/markdown',
```

Comment:
But readme is ReStructuredText!

Reply:
The ReST rendering problem on PyPI was caused by an unescaped `*.whl` in the changelog entry for 2.18.1.  It's already fixed in master.